boards/cc2538dk/dist/board.resc

```diff
@@ -27,4 +27,3 @@ macro reset
 """
 
 runMacro $reset
-start
```

Comment:
Why this change?

Reply:
because that doesn't work with the debugger otherwise. The startup sequence in the monitor is different. When using `emulate` alone, the start command is called using the `-e star` option of renode directly. When using `emulate debug*` targets, other commands are used to start the debug server and automatically start the emulated machine. If start is called before (from the .resc file), that doesn't work, the debug server is stuck and the gdb client cannot attach to it.

Also note that since #15501 and #15489 are merged, I have to adapt their configuration files as well.

Reply:
> Also note that since #15501 and #15489 are merged, I have to adapt their configuration files as well.

PR adapted